src/client.ts

```diff
@@ -4588,13 +4588,15 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      * @param {ReceiptType} receiptType The kind of receipt e.g. "m.read". Other than
      * ReceiptType.Read are experimental!
      * @param {object} body Additional content to send alongside the receipt.
+     * @param {boolean} unthreaded An unthreaded receipt will clear room+thread notifications
```

Comment:
`unthreaded` clearing both is strange

Maybe `threadAgnostic`?

Reply:
That is the terminology that was used in the MSC, https://github.com/matrix-org/matrix-spec-proposals/pull/3771